### PR TITLE
maybe null some refs which are causing hard dels problems

### PR DESCRIPTION
**Pull request by ShiftyRail**
I'm not really sure how important it is but I've noticed that every single `/obj/item` I was admin-deleting (by using the verb or proc-calling `qdel()`) wasn't actually properly deleted

This scares me, and while I was trying to fix (i couldn't), I noticed the following objects weren't properly nulled when `Destroy()` was called.

### CODE REVIEW

File: code/controllers/garbage.dm [hunk 1]
```diff
@@ -93,9 +93,8 @@ world/loop_checks = 0
 			to_chat(world, "we found [found]")
 			#endif
 
-
 			#ifdef GC_DEBUG
-			WARNING("gc process force delete [D.type]")
+			to_chat(world, "gc process force delete [D.type]")
```
Comment (jknpj) [inline code/controllers/garbage.dm]: Hitlerino pls

Comment (ShiftyRail) [reply code/controllers/garbage.dm]: `#ifdef GC_DEBUG` but fair

**Comment by Exxion** [comment]
>he doesn't know

Using a verb on something through the right-click menu causes BYOND to hold an internal reference to it for about five minutes
There's nothing we can do about it as far as I know

**Comment by ShiftyRail** [comment]
What a goddamn shame

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -95,9 +95,15 @@
 		src:holder = null
 	for(var/x in actions)
 		qdel(x)
+		x = null
```
Comment (DamianX) [inline code/game/objects/items.dm]: this seems useless

**Comment by DamianX** [comment]
Check travis logs.

**Comment by ShiftyRail** [comment]
came from a wrong understanding of the problem. Need to be redone

**Pull request closed without merging**